### PR TITLE
Portfolio: mobile menu

### PR DESCRIPTION
**Pull request by K0ppai**
In this pull request, I've implemented the following,

- Add linters configuration for HTML, CSS, and Javascript 
- Add the following features for

1.  When the user clicks (or taps) the hamburger button, the mobile menu appears.
2. When the user clicks (or taps) the close (X) button, the mobile menu disappears.
3. When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
4. When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

### CODE REVIEW

**Review by osamaashraf6 — COMMENTED** [review]
Hi @K0ppai 😊, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 🆘💀

- You have submitted a project, that no partners committed at the pull request, so this  💯milestone is pair programming, so please try to let your partners commit at the pull request to make it valid for reviewing 😊

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by K0ppai** [comment]
> Hi @K0ppai 😊,
> 
> While you made a great effort in this project, unfortunately, I cannot proceed to review your code.
> 
> ## Invalid Code Review Request 🆘💀
> * You have submitted a project, that no partners committed at the pull request, so this  💯milestone is pair programming, so please try to let your partners commit at the pull request to make it valid for reviewing 😊
> 
> Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏
> 
> Cheers, and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by K0ppai** [comment]
Hello @osamaashraf6 ,
The commit "Update script.js" is my partner commit. Somehow his profile is not showing.
image

File: script.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+hamburger.addEventListener('click', () => {
+  hamburger.classList.toggle('active');
+  navMenu.classList.toggle('active');
+});
+
+document.querySelectorAll('.nav-link').forEach((n) => n.addEventListener('click', () => {
+  hamburger.classList.remove('active');
+  navMenu.classList.remove('active');
+}));
```
Comment (KDlamini) [inline script.js:12]: Nice!!! 🔥🔥🔥
Minimal and effective. Well done!👌🏾

**Review by KDlamini — APPROVED** [review]
### Hi @K0ppai 🙋‍♀️,  

Excellent work! 💯 
You have done a great job implementing all the requirements for a responsive mobile menu. 🎊 🥳 
Fantastic job nailing it on your first review. I love the way you used animations in your application. It is very seamless and impressive!!! 👏🏾🎆🍾  
Your feature is complete! There is nothing else to say other than... it's time to merge it. :shipit: 
Congratulations!!!

## HIGHLIGHTS 🌟 

✔️ Good PR title and description
✔️ Mobile hamburger menu button is only visible on the mobile view
✔️ Mobile menu appears and closes as desired upon user interactions
✔️ The hamburger button and menu follow the design from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)
✔️ The JavaScript code is implemented following the [best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md).

# Status: Approved ✔️ 

oscars-standing-ovation
 
Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me @KDlamini  in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by KDlamini — APPROVED** [review]
### Hi @K0ppai 🙋‍♀️,  

Excellent work! 💯 
You have done a great job implementing all the requirements for a responsive mobile menu. 🎊 🥳 
Fantastic job nailing it on your first review. I love the way you used animations in your application. It is very seamless and impressive!!! 👏🏾🎆🍾  
Your feature is complete! There is nothing else to say other than... it's time to merge it. :shipit: 
Congratulations!!!

## HIGHLIGHTS 🌟 

✔️ Good PR title and description
✔️ Mobile hamburger menu button is only visible on the mobile view
✔️ Mobile menu appears and closes as desired upon user interactions
✔️ The hamburger button and menu follow the design from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)
✔️ The JavaScript code is implemented following the [best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md).

# Status: Approved ✔️ 

oscars-standing-ovation
 
Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me @KDlamini  in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**